hydroengine_service/main.py

```diff
@@ -1550,31 +1572,39 @@ def get_feature_info():
     r = request.get_json()
     image_id = r['imageId']
     bbox = r['bbox']
+    datasets = r.get('datasets', None)
     band = r.get('band', None)
     function = r.get('function', None)
     info_format = r.get('info_format', 'JSON')
+    if function == 'mosaic_elevation_datasets':
+        if not datasets:
+            msg = f'datsets list expected for function {function}'
+            raise error_handler.InvalidUsage(msg)
+        image = ee.Image(dgds_functions.mosaic_elevation_datasets(datasets).select('elevation'))
+    else:
+        image = ee.Image(image_id)
+        image_location_parameters = image_id.split('/')
+        source = ('/').join(image_location_parameters[:-1])
 
-    image = ee.Image(image_id)
-    image_location_parameters = image_id.split('/')
-    source = ('/').join(image_location_parameters[:-1])
+        data_params = dgds_functions.get_dgds_source_vis_params(source, image_id)
 
-    data_params = dgds_functions.get_dgds_source_vis_params(source, image_id)
+        if band:
+            band_name = data_params['bandNames'][band]
+            image = image.select(band_name)
+        if function:
+            assert (function in data_params.get('function', None)) or \
+                   (function == data_params['function'].get(band, None)), \
+                f'{function} not an option.'
 
-    if band:
-        band_name = data_params['bandNames'][band]
-        image = image.select(band_name)
-    if function:
-        assert (function in data_params.get('function', None)) or \
-               (function == data_params['function'].get(band, None)), \
-            f'{function} not an option.'
+        image = dgds_functions.apply_image_operation(image, function, data_params, band)
 
-    image = dgds_functions.apply_image_operation(image, function, data_params, band)
     image = image.rename('value')
 
     value = (
         image.sample(**{
             'region': ee.Geometry(bbox),
-            'geometries': True
+            'geometries': True,
+            'scale': 10
```

Comment:
is this always set to 10?

Reply:
No, this should depend on which scale we want to be accurate to, and also on the dataset. Now needed because elevation mosaic is of multiple resolutions, depending on sources included. added TODO, need to think about how to do this nicely.